zulipterminal/cli/run.py

```diff
@@ -157,6 +157,12 @@ def parse_args(argv: List[str]) -> argparse.Namespace:
         action="store_true",
         help="list all the organizations that you have configurations for, and exit",
     )
+    parser.add_argument(
+        "-n",
+        "--new-organization",
+        action="store_true",
+        help="login to a new organization",
+    )
```

Comment:
It doesn't quite fit within the remit of the zuliprc movement work, but it'd be good to have `-c` work with creating a zuliprc file from username/password, which I don't think it does right now?